src/atoms/NumberInputRow.test.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Enzyme, { shallow } from 'enzyme';
+import EnzymeAdapter from 'enzyme-adapter-react-16';
+import { findByTestAttr, checkProp } from '../utilities/testutils';
+import NumberInputRow from './NumberInputRow';
+
+Enzyme.configure({ adapter: new EnzymeAdapter() });
+
+const setup = (onInputChange, left, startValue) => shallow(
+  <NumberInputRow left={left} onInputChange={onInputChange} startValue={startValue} />,
+);
+
+test('renders without error', () => {
```

Comment:
So tests like this are usually titled `'it should render'`. The error part in there really confused me at first. I thought you were checking an error print area.